### ISSUE TITLE
some tests leak stdout, which is Bad.

### ISSUE THREAD

**Issue by ctb** [dib-lab/khmer#1125]
```
tests.test_script_arguments.test_create_countgraph_3 ...
** ERROR: khmer only supports k-mer sizes <= 32.

ok
```

and

```
tests.test_script_arguments.test_create_nodegraph_3 ...
** ERROR: khmer only supports k-mer sizes <= 32.

ok
```

and

```
tests.test_functions.test_check_file_status_kfile ... ERROR: Input file /var/folders/y6/s1r3q87933s6gjdrtmts1npc0000gq/T/khmertest_OSEr6M/thisfiledoesnotexist does not exist
Exiting
ok
```

and

```
tests.test_functions.test_check_file_status_kfile_force ... ERROR: Input file /var/folders/y6/s1r3q87933s6gjdrtmts1npc0000gq/T/khmertest_qMiKqT/thisfiledoesnotexist does not exist
ok
```

**Comment by ctb** [comment]
(yes, yes, mea culpa. but I didn't want to deal with it at the time :)

**Comment by bocajnotnef** [comment]
Ooh, low hanging fruit.

I'll nab this. (Mostly because that leak is annoying the crap outta me.)

**Comment by bocajnotnef** [comment]
Additionally, this test is leaking into the main directory (~/git/khmer):
`tests.test_scripts.test_extract_paired_reads_4_output_files ... FAIL`

**Comment by bocajnotnef** [comment]
Hmm. Strike my last, doesn't appear to be leaking anymore.. Weird.

**Comment by bocajnotnef** [comment]
Closing #1125, fixed by #1137
